app/app.go

```diff
@@ -97,203 +96,14 @@ func NewApp(
 		tkeys:          tkeys,
 	}
 
-	app.keeper.params = params.NewKeeper(
```

Comment:
is it possible to keep the common parts here and use the functions to add the tag-specific values?